app/helpers/search_helper.rb

```diff
@@ -14,4 +14,56 @@ def view_online(result)
 
     link_to 'View online', result['sourceLink'], class: 'button button-primary green'
   end
+
+  def view_record(record_id)
+    link_to 'View full record', record_path(id: record_id), class: 'button button-primary green'
+  end
+
+  def number_in_results(result)
+    result_index = 0
+    @results.each_with_index do |record, record_index|
+      result_index = record_index if record['timdexRecordId'] == result['timdexRecordId']
+    end
+    result_index + @pagination[:start]
+  end
+
+  # 'Coverage' and 'issued' seem to be the most prevalent types; 'coverage' is typically formatted as
+  # 'YYYY', whereas 'issued' comes in a variety of formats.
+  def parse_geo_dates(dates)
+    relevant_dates = if dates&.any? { |date| date['kind'] == 'Issued' }
+                       dates.select { |date| date['kind'] == 'Issued' }&.uniq
+                      elsif dates&.any? { |date| date['kind'] == 'Coverage' }
+                       dates.select { |date| date['kind'] == 'Coverage' }&.uniq
+                     end
+    return if relevant_dates.blank?
+
+    # Ignore dates of the same kind with unique values.
+    return if relevant_dates.length > 1
```

Comment:
Yeah, I'm pretty 🤷 with this, but I do think you're right that we should probably return something rather than nothing.

Reply:
I guess maybe it comes down to... is it better to display nothing or display something that is only a "partial truth". i.e. displaying 1999 when in reality it should say something like 1999/2005.

If we leave the current logic, let's update the comment to note that this might need further reflection and why.